### ISSUE TITLE
Search button lacking text for screen reader

### ISSUE THREAD

**Issue by BuddyinKC** [omeka-s-themes/freedom#54]
I didn't see this in the other logged issues so I thought I would log a new issue. We use DubBot and it is flagging the search box button for Freedom theme for lacking button text for screen readers. Here is the flagged code:

<button type="submit"><i class="fa fa-search"></i></button>

Let me know if you need any additional info. Thanks!

**Comment by kimisgold** [comment]
Thanks for alerting us. We have a fix ready that will be included in the next release of Freedom, which should coincide with the upcoming release of Omeka S 4.2.1.

**Comment by nelsonamaya82** [comment]
Closing as it's already fixed in the latest version v1.1.0. Thanks for reporting this issue.
